src/routes/productRouter.ts

```diff
@@ -69,4 +71,9 @@ router.get("/seller-get-products",userAuthorization(["seller"]),isSellerShopExis
 
 router.get("/user-get-products",isPaginated, productController.userGetProducts);
 
+router.get("/user-get-product/:id",isProductAvailable,productController.userViewSpecificProduct);
```

Comment:
Rename this `userViewSpecificProduct` into `userGetProduct` and `sellerViewSpecificProduct` to `sellerGetProduct`